lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1591,7 +1587,7 @@ class DxilRDATWriter : public DxilPartWriter {
 
   uint32_t AddShaderNodeInfo(const DxilModule &DM, llvm::Function &function,
                              const DxilEntryProps &entryProps,
-                             RuntimeDataFunctionInfo3 &funcInfo,
+                             RuntimeDataFunctionInfo2 &funcInfo,
```

Comment:
Where is RuntimeDataFunctionInfo3 defined? Should we remove the definition of RuntimeDataFunctionInfo3 too?

Reply:
It has been removed from the `RDAT_LibraryTypes.inl` where it was defined.  See deleted definition here:
https://github.com/microsoft/DirectXShaderCompiler/pull/5943/files/258e3f19ccab8788b6e80f0f620dbaa826c5a6f2#diff-2ea78813faaf946be274cbe2d57a3e174718a09199d9f90e5563aa713fa334adL552